mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```
@@ -363,6 +514,15 @@ static LogicalResult commonAttentionGemmElmtGemm(
     std::tie(a, b, c, out) = maybeSplitk.value();
   }
 
+  int64_t splitKVNum = splitKV.getInt();
+
+  // Grouped-Query Attention (GQA)
+  IntegerAttr numRepeatsGQA = nullptr;
```

Can we use FailureOr<> instead of `nullptr` for this?

---

numRepeatsGQA is a param we will use for GridwiseAttentionAccelOp, as it's optional (not needed for gemm+gemm for example), it needs to be null.